index.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>TearDown</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <nav class="navbar flex top-bar">
+      <div class="items logo"><img src="images/logo.png" alt="logo" /></div>
+      <div class="items1">
+        <h2><a href="#">Articles</a></h2>
+        <p><a href="#">Design & development</a></p>
+      </div>
+      <div class="items1">
+        <h2><a href="#">Events</a></h2>
+        <p><a href="#">Conferences & workshops</a></p>
+      </div>
+      <div class="items1">
+        <h2><a href="#">Jobs</a></h2>
+        <p><a href="#">Find work & employees</a></p>
+      </div>
+      <div class="items1">
+        <h2><a href="#">Membership</a></h2>
+        <p><a href="#">Webinars & early birds</a></p>
+      </div>
+      <div class="items1">
+        <h2><a href="#">Books</a></h2>
+        <p><a href="#">Physical & digital books</a></p>
+      </div>
+      <div class="items1"><i class="fa fa-search"></i> Topics</div>
+    </nav>
+
+    <div class="articles flex lower-bar">
+      <p class="latest-articles">Don’t Miss These Latest Articles</p>
+      <div class="items3">
+        <h1>Art Direction For The Web Using CSS Shapes</h1>
+      </div>
+      <div class="items2">
+        <h1>Privacy UX: Better Cookie Consent Experiences</h1>
+      </div>
+      <div class="items3">
+        <h1>Understanding Subresource Integrity</h1>
+      </div>
+      <div class="items2">
+        <h1>Digging Into The Display Property: The Two Values Of Display</h1>
+      </div>
+    </div>
+    <div class="membership top-bar items4"><h1>Membership</h1></div>
+
+    <div class="latest-posts flex lower-bar">
+      <div class="items3">
+        <h2>Lorem Ipsum</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+          condimentum fermentum lacus at tempus. Vivamus dapibus consectetur
+          tincidunt. Donec quis posuere velit, vel congue elit. In faucibus
+          faucibus
+        </p>
+      </div>
+      <div class="items2">
+        <h2>Lorem Ipsum</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+          condimentum fermentum lacus at tempus. Vivamus dapibus consectetur
+          tincidunt. Donec quis posuere velit, vel congue elit. In faucibus
+          faucibus eget molestie elit nibh ac odio.
+        </p>
+      </div>
+      <div class="items1">
```

Comment:
For this type of articles, you should have defined  3 taking one-third of the total width. (A row of three articles)